server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessageListMethodTest.java

```diff
@@ -2285,6 +2288,183 @@ public void getMessageListShouldReturnTwoMessagesWhenCopiedAtOnceViaIMAP() throw
+
+        String inboxMessageId = latestMessageId(userAccessToken, Role.INBOX);
+        with()
+            .header("Authorization", userAccessToken.serialize())
+            .body("[[\"getMessages\", {\"ids\": [\"" + inboxMessageId + "\"]}, \"#0\"]]")
+        .when()
+            .post("/jmap")
+        .then()
+            .statusCode(200)
+            .body(NAME, equalTo("messages"))
+            .body(ARGUMENTS + ".list", hasSize(1))
+            .body(ARGUMENTS + ".list[0].textBody", equalTo(messageContent));
+    }
+
+    @Test
+    public void sendShouldNotCreateNewUserOrMailboxOfAliasAddressWhenDomainAliasMapping() throws Exception {
+        dataProbe.addDomain("domain1.com");
+        dataProbe.addDomain("domain2.com");
+        dataProbe.addUser("user@domain1.com", "password");
+        dataProbe.addUser("sender@domain1.com", "password");
+
+        // domain2 as the source, domain1 as the destination
+        dataProbe.addDomainAliasMapping("domain2.com", "domain1.com");
+
+        AccessToken userAccessToken = authenticateJamesUser(baseUri(jmapServer), "user@domain1.com", "password");
+        AccessToken senderAccessToken = authenticateJamesUser(baseUri(jmapServer), "sender@domain1.com", "password");
+        String messageContent = "content content";
+        String sendMessageBody =
+            "[" +
+            "  [" +
+            "    \"setMessages\"," +
+            "    {" +
+            "      \"create\": { \"creationId1337\" : {" +
+            "        \"from\": { \"name\": \"Sender\", \"email\": \"sender@domain1.com\"}," +
+            "        \"to\": [{ \"name\": \"User\", \"email\": \"user@domain2.com\"}]," +
+            "        \"subject\": \"Thank you for joining example.com!\"," +
+            "        \"textBody\": \"" + messageContent + "\"," +
+            "        \"mailboxIds\": [\"" + getOutboxId(senderAccessToken) + "\"]" +
+            "      }}" +
+            "    }," +
+            "    \"#0\"" +
+            "  ]" +
+            "]";
+        given()
+            .header("Authorization", senderAccessToken.serialize())
+            .body(sendMessageBody)
+        .when()
+            .post("/jmap")
+        .then()
+            .statusCode(200)
+            .body(NAME, equalTo("messagesSet"))
+            .body(ARGUMENTS + ".notCreated", aMapWithSize(0))
+            .body(ARGUMENTS + ".created", aMapWithSize(1));
+
+        calmlyAwait
+            .pollDelay(Duration.FIVE_HUNDRED_MILLISECONDS)
+            .atMost(30, TimeUnit.SECONDS).untilAsserted(() -> assertThat(latestMessageId(userAccessToken, Role.INBOX)).isNotNull());
+
+        assertThat(dataProbe.listUsers())
+            .doesNotContain("user@domain2.com");
+        assertThat(mailboxProbe.listUserMailboxes("user@domain2.com"))
+            .isEmpty();
+    }
+
+
+    @Test
+    public void sendShouldSaveToAsTheAliasAddressWhenDomainAliasMapping() throws Exception {
```

Comment:
The name of yourtest (**send**) rather suggest this is a **SetMessage** related test.

Please move this to the right class...